utilities/ImpactUtilities.groovy

```diff
@@ -442,6 +445,8 @@ def calculateChangedFiles(BuildResult lastBuildResult, boolean calculateConcurre
 				(file, mode) = fixGitDiffPath(file, dir, false, mode)
 				deletedFiles << file
 				if (props.verbose) println "**** $file"
+			} else {
+				if (props.verbose) println "**** $file is deleted, but not in build scope. See excludeFileList configuration. No follow-up processing."
```

Comment:
I'm picky here, but could we have a consistent wording -> "but is excluded from build scope"